### ISSUE TITLE
Mobile view to scroll one image at a time

### ISSUE THREAD

**Issue by ezhang816** [OrionMD/pacsbin-community#23]
Hi Jason, 

On the desktop view, we can use up and down arrows to scroll one slice at a time. There is no way to do this on mobile view unless to drag very slowly, which is difficult.

Thanks!

**Comment by jmhmd** [comment]
This is true. I'm not sure yet what the ideal interface for this would be - buttons to scroll would take up valuable screen real-estate. Any thoughts? Maybe a button to toggle "Slow scroll" mode?

**Comment by ezhang816** [comment]
I'd be happy with any method that allows me to scroll one slice at a time, either with arrow buttons or a slow scroll mode. Or perhaps scroll by one slice if you can detect that the user is dragging their finger slowly?

**Comment by jmhmd** [comment]
Here is a potential concept which I think might work well for small mobile screens where interaction will be less deep than on a desktop. Adding a bottom menu for buttons to toggle case list, toggle notes, and scroll by single image. Not sure if this is optimal but a good starting point I think:

<img src="https://github.com/user-attachments/assets/df33be98-8b0a-4dd2-9263-135a3a20ac29" width="300">

**Comment by jmhmd** [comment]
Fixed in 4069d93cda4b14c6c7368727afcb15ecccefac42
